VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddPostFragment.java

```diff
@@ -2,6 +2,8 @@
 
 import android.os.Bundle;
 import android.support.annotation.StringRes;
+import android.support.v4.view.ViewCompat;
+import android.support.v7.widget.AppCompatSpinner;
```

Comment:
This import doesn't seem to be used, and since `AppCompatSpinner` probably doesn't define any API over what `Spinner` provides, it probably doesn't need to be referred to in the code.